src/kpatch_elf.c

```diff
@@ -966,3 +967,35 @@ int kpatch_elf_load_kpatch_info(struct object_file *o)
 	kpdebug("failed\n");
 	return -1;
 }
+
+int kpatch_unpatch_find_address(struct object_file *o, unsigned long address)
```

Comment:
This function have nothing to do with the unpatch. It is only called while unpatching. Please name it accordingly to what it does, and not the use.